golang/cmd/mqtt-to-postgresql/database.go

```diff
@@ -227,6 +227,24 @@ func GetLatestParentUniqueProductID(aid string, assetID uint32) (uid int32, succ
 	return
 }
 
+func CheckIfProductExists(productId int32) (exists bool) {
+	txn, err := db.Begin()
```

Comment:
Add caching!

Reply:
Resolved in 9d5ccef27a35d24a2d48fc731c48dd17e6648a25